lib/tgspam/openai.go

```diff
@@ -22,6 +22,7 @@ type openAIChecker struct {
 
 // OpenAIConfig contains parameters for openAIChecker
 type OpenAIConfig struct {
+	BaseURL           string
```

Comment:
is there a reason why we even need this new field here? from what I see, the only place you should set it is in main.go makeDetector (which you did), but I can't understand why it is added here too.

Reply:
As I am not proficient in go - it was trial and error process.
If you can build the app without this change, than it is not needed indeed.

Reply:
I can't
```
# github.com/umputun/tg-spam/app
.\main.go:430:4: unknown field BaseURL in struct literal of type tgspam.OpenAIConfig
```

Reply:
Fixed